pyanaconda/modules/storage/bootloader/efi.py

```diff
@@ -197,13 +203,70 @@ def write_config(self):
         super().write_config()
 
 
+class EFISystemdBoot(EFIBase, SystemdBoot):
+    """EFI Systemd-boot"""
+    _packages_common = ["efibootmgr", "systemd-udev", "systemd-boot", "sdubby"]
```

Comment:
sdubby does not exist

Reply:
@jlinton should we remove the `sdubby` package or is this something special?

Reply:
The package should be mandatory, but it is still in review (https://bugzilla.redhat.com/show_bug.cgi?id=2134972). See the comments above.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=2134972
It is under review

Reply:
Formally it is under review. But I think that if the review is supposed to be more than just a rubber-stamp procedure, then IMO this particular package should not pass it. It provides a concentrated dose of bad technical choices and super super brittle workarounds for shortcomings in other packages. The good thing is that _all_ of those other packages are under our control, so we can fix things: anaconda, grub2, kernel-install, kernel scriptlets, systemd-boot. We don't need a combination of rpm packaging, bash, python, and awk to serve as glue between those other packages.